src/openvic-simulation/pop/Pop.cpp

```diff
@@ -319,15 +319,16 @@ bool PopManager::load_pop_into_vector(
 
 bool PopManager::generate_modifiers(ModifierManager& modifier_manager) const {
 	bool ret = true;
+
 	for (Strata const& strata : get_stratas()) {
-		const auto strata_modifier = [&modifier_manager, &ret, &strata](std::string_view suffix, bool positive_good) -> void {
+		const auto strata_modifier = [&modifier_manager, &ret,
+									  &strata](std::string_view suffix, bool is_positive_good) -> void {
```

Comment:
```suggestion
		const auto strata_modifier = [&modifier_manager, &ret, &strata](
			std::string_view suffix, bool is_positive_good
		) -> void {
```
Again, could you remove the extra indentation, I have no idea why the formatter thinks it should go to a new line in the middle of the lambda capture list

Reply:
Probably because the line became too long. Honestly I struggle configuring vs code formatter for c++.
I'm used to Resharper and VS2022 formatting C#.